behaviortree.tf

```diff
@@ -4,6 +4,7 @@ locals {
   ]
   behaviortree_repositories = [
     "behaviortree_cpp-release",
+    "behaviortree_cpp-release-v4",
```

Comment:
```suggestion
    "behaviortree_cpp-release",
    "behaviortree_cpp_v4-release",
```

Reply:
I'd suggest this format change to ensure the package name and release repo name match the standard convention.